taiga_contrib_ldap_auth/services.py

```diff
@@ -62,17 +67,29 @@ def ldap_login_func(request):
     # (or any other attribute)
     login_input = request.DATA.get('username', None)
     pass_input = request.DATA.get('password', None)
-
-    # TODO: sanitize before passing to LDAP server?
-    username, email, full_name = connector.login(login = login_input,
-                                                 password = pass_input)
-
-    user = register_or_update(username = username,
-                              email = email,
-                              full_name = full_name)
-
-    data = make_auth_response_data(user)
-    return data
+    try:
+        # TODO: make sure these fields are sanitized before passing to LDAP server!
+        username, email, full_name = connector.login(login = login_input, password = password_input)
+    except connector.LDAPUserLoginError as ldap_error:
```

Comment:
> I don't think explicitly syncing is a good idea as this kinda defeats the advantages of having LDAP in the first place.

Agreed.

Let's see... Why is there a need for fallback auth via "normal" in the first place?

https://github.com/ensky/taiga-contrib-ldap-auth/issues/49 outlines one use case, but IMO they should be implementing LDAP registration instead.

The only "valid" reason I can think of is allowing those "admin" users that were created on a Taiga deployment prior to LDAP integration to still log in. Those can likely be specified as "exceptions to the rule" in the config.